mitodl/ol-infrastructure
# config: Use self hosted logos (xPRO)

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/3203

### Description (What does it do?)

We have been using logos directly from GitHub in the past that has led to inconsistency issues being seen in Production. This PR now makes us use the self hosted logos.


### How can this be tested?

Once deployed:

-  Check the relevant CI or RC or Prod course Learning MFE page e.g. `/learn` pages and notice that the logos shown on that page use the self hosted URLs as mentioned in https://github.com/mitodl/hq/issues/3203.
- Open any lecacy page. e.g. Prograss page or Instructor dashboard and check that the logo URLs are self hosted ones.
- Open studio and check the logo use self hosted URLs.
